packages/history-sdk/package.json

```diff
@@ -23,11 +23,11 @@
     "./dist/commercetools-history-sdk.esm.js": "./dist/commercetools-history-sdk.browser.esm.js"
   },
   "dependencies": {
-    "@commercetools/sdk-client": "^2.1.1",
+    "@commercetools/sdk-client": "^2.1.2",
     "@commercetools/sdk-middleware-auth": "^6.0.4",
     "@commercetools/sdk-middleware-http": "^6.0.4",
     "@commercetools/sdk-middleware-logger": "^2.1.1",
```

Comment:
as i just go through the cahnges

shouldn't this be changed to the sdk-client-v2 ?

Also the different middlewares are already part of the sdk-client-v2 so not needed anymore here. Same applies to the other sdk packages